zerver/tests/test_exception_filter.py

```diff
@@ -0,0 +1,52 @@
+from django.test import RequestFactory
+from django.views.debug import ExceptionReporter
+
+from zerver.filters import ZulipExceptionReporterFilter
+from zerver.lib.test_classes import ZulipTestCase
+
+
+class TestExceptionFilter(ZulipTestCase):
+    def test_zulip_filter_masks_sensitive_post_data(self) -> None:
+        rf = RequestFactory()
+        request = rf.post(
+            "/test",
+            {
+                "password": "sneaky",
+                "api_key": "abc123",
+                "content": "secret msg",
+                "realm_counts": "private",
+                "installation_counts": "private",
+            },
+        )
+
+        filt = ZulipExceptionReporterFilter()
+        cleaned = filt.get_post_parameters(request)
+
+        for var in [
+            "password",
+            "api_key",
+            "content",
+            "realm_counts",
+            "installation_counts",
```

Comment:
Can you add comments here explaining for each of these details what rule results in them being filtered?

I guess this is just the explicit `filtered_vars` field?